assignment_2.java

```diff
@@ -0,0 +1,144 @@
+/***
+ *When the program is run, the user will first enter a string. After that, the program should
+ask the user which operation they want to execute on the string. The available operations are
+listed below: Append, CountWords, Replace, isPalindrome, Splice, Split,
+MaxRepeatingCharacter, Sort, Shift, and Reverse. Each method should perform its
+respective operation based on the user's input.
+Owner : Pratul_Agarwal
+Date of creation : 04/09/2024
+ */

```

Comment:
No issues here, but make sure the package name (assignment_java_programming) matches your project structure.